packages/truffle-compile-vyper/test/test_compiler.js

```diff
@@ -1,60 +1,127 @@
     quiet: true,
-    all: true,
+    all: true
   });
 
-  it('compiles vyper contracts', function (done) {
-    compile.all(config, function (err, contracts, paths) {
-      assert.equal(err, null, 'Compiles without error');
-
-      paths.forEach(function (path) {
-        assert(['.vy', '.v.py', '.vyper.py'].some(extension => path.indexOf(extension) !== -1), 'Paths have only vyper files');
+  it("compiles vyper contracts", function(done) {
+    compile.all(config, function(err, contracts, paths) {
+      assert.equal(err, null, "Compiles without error");
+
+      paths.forEach(function(path) {
+        assert(
+          [".vy", ".v.py", ".vyper.py"].some(
+            extension => path.indexOf(extension) !== -1
+          ),
+          "Paths have only vyper files"
+        );
       });
 
       const hex_regex = /^[x0-9a-fA-F]+$/;
 
       [
         contracts.VyperContract1,
         contracts.VyperContract2,
-        contracts.VyperContract3,
+        contracts.VyperContract3
       ].forEach((contract, index) => {
-        assert.notEqual(contract, undefined, `Compiled contracts have VyperContract${index + 1}`);
-        assert.equal(contract.contract_name, `VyperContract${index + 1}`, 'Contract name is set correctly');
-
-        assert.notEqual(contract.abi.indexOf('vyper_action'), -1, 'ABI has function from contract present');
-
-        assert(hex_regex.test(contract.bytecode), 'Bytecode has only hex characters');
-        assert(hex_regex.test(contract.deployedBytecode), 'Deployed bytecode has only hex characters');
-
-        assert.equal(contract.compiler.name, 'vyper', 'Compiler name set correctly');
+        assert.notEqual(
+          contract,
+          undefined,
+          `Compiled contracts have VyperContract${index + 1}`
+        );
+        assert.equal(
+          contract.contract_name,
+          `VyperContract${index + 1}`,
+          "Contract name is set correctly"
+        );
+
+        assert.notEqual(
+          contract.abi.indexOf("vyper_action"),
+          -1,
+          "ABI has function from contract present"
+        );
+
+        assert(
+          hex_regex.test(contract.bytecode),
+          "Bytecode has only hex characters"
+        );
+        assert(
+          hex_regex.test(contract.deployedBytecode),
+          "Deployed bytecode has only hex characters"
+        );
+
+        assert.equal(
+          contract.compiler.name,
+          "vyper",
+          "Compiler name set correctly"
+        );
       });
 
       done();
     });
   });
 
-  it('skips solidity contracts', function (done) {
-    compile.all(config, function (err, contracts, paths) {
-      assert.equal(err, null, 'Compiles without error');
+  it("compiles with source_map option", function(done) {
+    // this test pass when vyper 0.1.0-b9 or higher
+    exec("vyper --version", function(err, stdout) {
+      let version;
+      if (!err) version = stdout.trim();
+
+      if (version === "0.1.0-b9") {
+        config.merge({
```

Comment:
So then this section needs to be updated to:

```suggestion
        config.merge({
            compilers: {
              vyper: {
                sourceMap: true
              }
            }
        });
```

Reply:
Extra note on this: let's get `sourceMap` or `source_map` inside a `vyper.settings` namespace:
```javascript
  compilers: {
    vyper: {
      settings: {
        sourceMap: true
      }
    }
  }
 